securedrop/template_filters.py

```diff
@@ -22,19 +23,20 @@ def _relative_timestamp(dt):
         delta.microseconds + (delta.seconds +
                               delta.days * 24 * 3600) * 1e6) / 1e6
     if diff < 45:
-        return '{} second{}'.format(int(diff), '' if int(diff) == 1 else 's')
+        return ngettext('{n} second', '{n} seconds', int(diff)).format(
+            n=int(diff))
     elif diff < 90:
-        return 'a minute'
+        return gettext('a minute')
     elif diff < 2700:
-        return '{} minutes'.format(int(max(diff / 60, 2)))
+        return gettext('{n} minutes').format(n=int(max(diff / 60, 2)))
```

Comment:
For languages that have a dual form (e.g. Arabic), we'll need to do something else here for the "2 minute" case. But let's do this in a followup.